djinni/jni/djinni_support.hpp

```diff
@@ -23,8 +23,8 @@
 #include <string>
 #include <vector>
 
-#include "../proxy_cache_interface.hpp"
-#include "../djinni_common.hpp"
+#include "djinni/proxy_cache_interface.hpp"
+#include "djinni/djinni_common.hpp"
```

Comment:
Nice! I see `..` in include directives as a smell (even though we had some weird stuff in my previous PRs 😅)